vortex-array/src/arrays/bool/compute/is_constant.rs

```diff
@@ -5,7 +5,11 @@ use crate::compute::{IsConstantKernel, IsConstantKernelAdapter, IsConstantOpts};
 use crate::register_kernel;
 
 impl IsConstantKernel for BoolVTable {
-    fn is_constant(&self, array: &BoolArray, _opts: &IsConstantOpts) -> VortexResult<Option<bool>> {
+    fn is_constant(&self, array: &BoolArray, opts: &IsConstantOpts) -> VortexResult<Option<bool>> {
+        if opts.is_constant() {
```

Comment:
Could actually allow len <= 64 to be constant time, since we're just comparing the boolean buffer as a single u64 to the zero value.

Reply:
Sure